### PR TITLE
Added POST /platforms to README

### PR DESCRIPTION
**Pull request by Fenntasy**
Added an entry to the README that was forgotten

### CODE REVIEW

**Review by AGuyNextDoor — DISMISSED** [review]
Wow great code !

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ yarn db:reset
 ### Rest client
 
 - `GET /platforms`
+- `POST /platforms`
 - `GET /platforms/:slug`
 - `GET /games/`
 - `GET /games/:slug`
```
Comment (AGuyNextDoor) [inline README.md:58]:
```suggestion
- `GET /platforms`;
- `POST /platforms`;
- `GET /platforms/:slug`;
- `GET /games/`;
- `GET /games/:slug`;
```

**Review by AGuyNextDoor — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ yarn db:reset
 ### Rest client
 
 - `GET /platforms`
```
Comment (Fenntasy) [inline README.md:54]: why GET?

Comment (Fenntasy) [reply README.md:54]: because why not!

Comment (AGuyNextDoor) [reply README.md:54]: Because I said so !

File: README.md (hunk 1 above)
Comment (Fenntasy) [reply README.md:58]: This is more HTTP listing than javascript.

File: README.md (hunk 2 above)
Comment (AGuyNextDoor) [reply README.md:54]: Are you ignoring me?

**Pull request merged**